lib/FieldType/Mapper/TextLineFormMapper.php

```diff
@@ -59,20 +67,33 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
             );
     }
 
+    /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
     public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
     {
         $fieldDefinition = $data->fieldDefinition;
+        $validatorConfiguration = $fieldDefinition->getValidatorConfiguration();
```

Comment:
If you create a private method below, I think you can call `getValidatorConfiguration()` directly from the method call and drop this line.